src/engine/drpc_client.c

```diff
@@ -171,6 +174,26 @@ drpc_notify_ready(void)
 	if (rc != 0)
 		goto out_uri;
 
+	nr_sec_uris = crt_get_nr_secondary_providers();
+	if (nr_sec_uris > 0) {
+		D_ALLOC_ARRAY(sec_uris, nr_sec_uris);
+		if (sec_uris == NULL)
+			goto out_uri;
+		for (i = 0; i < nr_sec_uris; i++) {
+			rc = crt_self_uri_get_secondary(0, &sec_uris[i]);
```

Comment:
should the first parameter to the call always be 0?

Reply:
Good catch. That should be `i`.